src/MessageBird/Resources/Conversation/Messages.php

```diff
@@ -123,6 +124,25 @@ public function getList($conversationId, $parameters = array())
         return $this->processRequest($body);
     }
 
+    public function readMessage($messageId, $parameters = array()) {
```

Comment:
In other modules this is called `Base::read()`. Could we use the same naming here to keep it consistent?